app/views/employees/_form.html.erb

```diff
@@ -1,5 +1,11 @@
 <%= simple_form_for @employee do |form| %>
-  <%= form.input :slack_username %>
+  <%= form.input :slack_username,
+                 required: true,
+                 pattern: '^[^@]\w+[^ ]$',
+                 input_html: {
+                   title: I18n.t('employees.errors.slack_username')
```

Comment:
hey! yes that is weird. I just pulled down your branch and played with it for a few mins, saw same thing.

my guess is something similar to this: https://github.com/thoughtbot/factory_girl/issues/714

> webmock doesn't actually trigger until you're in a spec
 
while I'd love to get to the bottom of this, giving this a second look makes me think we should take a slightly different approach. 

rather than adding this check as an active record validation, which means it will be checked every time a user is created, maybe we can add the check as a method in `Employee` so it only gets hit when we use that method (in the controller, for instance).

My reasoning is similar to this: http://stackoverflow.com/a/4830883/1019369

Hitting an API is expensive so we don't want to do it each time we create an `employee` record. Well we do....kind of. But having this check in the AR validation means we are going to be hitting this validation even when we are creating test `employee` for totally unrelated features. 

With a method rather than the AR validation we can still have a very similar controller cleanup to what you have here. I pulled down your branch and got something working, let me know what you think:

https://github.com/18F/dolores-landingham-bot/commit/efe7847fbcad025b3d5d5643c460ed05c2b9d4d3